.travis.yml

```diff
@@ -147,8 +147,9 @@ after_success:
       lcov --version
       pwd && ls -laF . /usr/bin/gcov*
 
-      cd $BUILD_ROOT && pwd
-      find ~/ -name "*gcno" -or -name "*gcda"
+      # cd $BUILD_ROOT && pwd
+      cd $SOURCE_ROOT && pwd
+      # find ~/ -name "*gcno" -or -name "*gcda"
```

Comment:
I like that find is commented out now.

Reply:
Yeah, that was a debugging thing, and it was spamming up the logs.